app-modules/project/src/Filament/Resources/ProjectResource/Pages/ManageMilestones.php

```diff
@@ -0,0 +1,126 @@
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Aiding App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AidingApp\Project\Filament\Resources\ProjectResource\Pages;
+
+use AidingApp\Project\Filament\Resources\ProjectResource;
+use AidingApp\Project\Models\ProjectMilestone;
+use App\Features\ProjectMilestoneFeature;
+use App\Filament\Tables\Columns\IdColumn;
+use Filament\Forms\Components\Select;
+use Filament\Forms\Components\Textarea;
+use Filament\Forms\Components\TextInput;
+use Filament\Forms\Form;
+use Filament\Resources\Pages\ManageRelatedRecords;
+use Filament\Tables\Actions\BulkActionGroup;
+use Filament\Tables\Actions\CreateAction;
+use Filament\Tables\Actions\DeleteAction;
+use Filament\Tables\Actions\DeleteBulkAction;
+use Filament\Tables\Actions\EditAction;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Table;
+
+class ManageMilestones extends ManageRelatedRecords
+{
+    protected static string $resource = ProjectResource::class;
+
+    protected static string $relationship = 'milestones';
+
+    public static function getNavigationLabel(): string
+    {
+        return 'Milestones';
+    }
+
+    public static function canAccess(array $arguments = []): bool
+    {
+        $user = auth()->user();
+
+        return ProjectMilestoneFeature::active() && $user->can('viewAny', [ProjectMilestone::class, $arguments['record']]);
+    }
+
+    public function form(Form $form): Form
+    {
+        return $form
+            ->schema([
+                TextInput::make('title')
+                    ->required()
+                    ->maxLength(255),
+                Textarea::make('description')
+                    ->required()
+                    ->maxLength(65535),
+                Select::make('status_id')
+                    ->searchable()
+                    ->preload()
+                    ->required()
+                    ->relationship('status', 'name'),
+                Select::make('created_by_id')
+                    ->label('Created By')
+                    ->relationship('createdBy', 'name'),
```

Comment:
This shouldn't be something that is selected, it should just be auto set by an observer one this Model